packages/new-deploy/src/DeployContext.ts

```diff
@@ -137,6 +149,7 @@ export default class DeployContext extends MultiProvider<config.Domain> {
 
     await core.deployHome();
     await core.deployGovernanceRouter();
+    // await Promise.all([core.deployHome(), core.deployGovernanceRouter()]);
```

Comment:
delete pls :)

Reply:
fixed in [92dde9a](https://github.com/nomad-xyz/monorepo/pull/81/commits/92dde9a14ab7121d4d2abadcc316003c217d4c57)